### PR TITLE
snap: update to latest version

### PR DESCRIPTION
**Pull request by Darnor**

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: cevelop
 base: core18
-adopt-info: cevelop
+version: 1.12.1-201907101456
```
Comment (AlanGriffiths) [inline snap/snapcraft.yaml:3]: I want to drive the version from the downloaded archive, so I don't like this approach.

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -9,15 +9,15 @@ description: |
 grade: stable
 confinement: classic
 
+architectures:
+  - build-on: amd64
+
```
Comment (AlanGriffiths) [inline snap/snapcraft.yaml:14]: Yes. I missed that it is an x86_64 archive.

File: snap/snapcraft.yaml [hunk 3]
```diff
@@ -9,15 +9,15 @@ description: |
...
 parts:
   cevelop:
     plugin: dump
-    # Source ought to be some "current" download, maybe we can add a sig check?
-    source: https://www.cevelop.com/cevelop/downloads/cevelop-1.11.1-201902151304-linux.gtk.x86_64.tar.gz
+    source: https://www.cevelop.com/cevelop/downloads/$SNAPCRAFT_PROJECT_NAME-$SNAPCRAFT_PROJECT_VERSION-linux.gtk.x86_64.tar.gz
```
Comment (AlanGriffiths) [inline snap/snapcraft.yaml:18]: See above. I should probably try scraping https://www.cevelop.com/download/index.html Something along the lines of:
```plain
curl -s https://www.cevelop.com/download/ | sed -n 's!^.*href="/cevelop/downloads/cevelop-\([0-9.-]*\)-linux.gtk.x86_64.tar.gz.*!\1!p' -
```

**Review by AlanGriffiths — CHANGES_REQUESTED** [review]

File: snap/snapcraft.yaml (hunk 3 above)
Comment (Darnor) [reply snap/snapcraft.yaml:18]: Sounds good. I'll update the PR.

**Comment by AlanGriffiths** [comment]
@Darnor, I think this achieves both of our goals: https://github.com/AlanGriffiths/cevelop-snap/pull/2

**Comment by Darnor** [comment]
> @Darnor, I think this achieves both of our goals: #2

Looks good. :+1:, thank you for your work.

**Pull request closed without merging**